run_dir/design/base.html

```diff
@@ -51,6 +51,7 @@
                         <li><a href="/reads_vs_qv">Reads vs QV</a></li>
                         <li><a href="/production/cronjobs">Cronjobs List</a></li>
                         <li><a href="/quotas">Quotas</a></li>
+                        <li><a href="/clusters_per_lane">Clusters per lane</a></li>
                         <li class="depreciated_page divider"></li>
                         <li id="depreciated_header"><a href="#">Depreciated Pages</a></li>
```

Comment:
untrustable isn't a word sorry ;) Apparently [I'm not the only one to make this mistake](http://english.stackexchange.com/questions/45295/why-is-there-confusion-between-depreciated-and-deprecated), which is nice.

s/depreciated/deprecated/g

Sorry for my typos everyone :disappointed: